wurst/systems/modes/GameMode.wurst

```diff
@@ -376,7 +375,7 @@ init
         mode.message(ENERGY_COLOR + "Reverse Famine|r has been enabled. More animals spawn and they give more food.")
 
     new GameMode("raise-food-limit", "rfl", "More animals and fish can be on the map at the same time", false) (mode, args) ->
-        udg_ANIMAL_MAX = 480
```

Comment:
Let's use `gameConfig.setMaxAnimals(int val)` or at the very least `gameConfig.maxAnimals`

Reply:
Changed